src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraVolumeMapperV1.kt

```diff
@@ -15,13 +15,19 @@ class AuroraVolumeMapperV1(val applicationFiles: List<AuroraConfigFile>) {
     val mountHandlers = createMountHandlers()
     val configHandlers = applicationFiles.findConfigFieldHandlers()
 
-    val handlers = configHandlers + mountHandlers + listOf(AuroraConfigFieldHandler("secretVault"))
+    val handlers = configHandlers + mountHandlers + listOf(
+            AuroraConfigFieldHandler("secretVault"),
+            AuroraConfigFieldHandler("secretVaultKeys")
```

Comment:
Dette er definitvt en enkel og god variant, men bør vi heller legge til støtte for "utvidet" konfig som vi har f.eks. for database og cert? Eks:

...
"secretVault": "somevault"
...
"secretVault": {
  "name": "somevault",
  "keys": ["password", "secret"]
}

Litt mer omfattende løsning, men da blir det til gjengjeld noe enklere å f.eks. støtte excludeKeys ell. senere?

Reply:
hmm, ser at det vil være bedre for senere utvidelser, men vi må vel støtte begge formater da, for bakoverkompatibilitetens skyld om ikke annet. 
Vil det være bedre å gjøre den utvidelsen nå, eller å gjøre det når/hvis behovet oppstår?

Reply:
Jeg tror dette må bli @bjartek s beslutning, men jeg tror jeg heller mot å gå for "utvidet" config personlig. Kanskje ikke først og fremst for å legge til rette for fremtidige utvidelser, men for å følge de konvensjonene som allerede er på plass.

Reply:
Vi bruker utvidet config noen plasser det stemmer det, så da burde vi kanskje gjøre det her også. Hvis du ser under mountHandlers så ser du hvordan man gjør utvidede handlers. Eller så kan jeg fikse det for deg. 

Det er egentlig litt min feil at det er secretVaultKeys fordi det var det jeg foreslo, men jeg er enig med BentAndre at det er en ryddigere løsning å støtte utvidet (og enkel) konfig som vi gjør andre plasser.